src/gui/redux/docgen/docgen.selectors.js

```diff
@@ -1,12 +1,6 @@
 import { createSelector } from 'reselect'
-import docgen from './docgen'
+import { docgen } from '../store'
 
-const getDocgen = state => docgen.get()(state) || {}
-
-const getProps = createSelector(getDocgen, data => data.props || {})
-
-const getPropsKeys = createSelector(getProps, props => Object.keys(props))
-
-const getProp = name => createSelector(getProps, props => props[name])
-
-export { getDocgen, getProps, getPropsKeys, getProp }
+const getProps = state => docgen.get()(state).props
```

Comment:
`getPropsKeys` selector do : `Object.keys(undefined)` because `data.props` is undefined, then you get this kind of error :
```
Uncaught TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
```

You can see it in the GUI, when you click to 'documentation' button, then you come back clicking to 'workbench' button